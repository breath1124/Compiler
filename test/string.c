int main()
{
    string s = "a";
    print s;
}

                                                        

