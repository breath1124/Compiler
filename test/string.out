24 19 0 5 25 13 28 61 12 13 11 22 21 1